client/lib/plans/features-list.js

```diff
@@ -239,9 +241,13 @@ export const FEATURES_LIST = {
 		getSlug: () => FEATURE_ALL_PREMIUM_FEATURES,
 		getTitle: () => i18n.translate( 'All Premium features' ),
 		getDescription: () => {
-			return i18n.translate(
-				'Including advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
-			);
+			return isEnabled( 'themes/premium' )
+				? i18n.translate(
+						'Including premium themes, advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Including unlimited premium themes, advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/themes-premium-ref-to-start-plans)